src/lm/arpa-file-parser.h

```diff
@@ -86,9 +91,35 @@ class ArpaFileParser {
   void Read(std::istream &is, bool binary);
 
   /// Parser options.
-  const ArpaParseOptions& Options() const { return options_; }
+  const ArpaParseOptions &Options() const { return options_; }
```

Comment:
It actually does--from my experience it is far easier to spot errors in uniform code. I'll change  affected lines in this patch.